packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -77,6 +141,22 @@ export default function EditorStyles( { styles } ) {
 			{ /* Use an empty style element to have a document reference,
 			     but this could be any element. */ }
 			<style ref={ useDarkThemeBodyClassName( styles ) } />
+			{ _styles.map(
```

Comment:
Where do you think these styles should load? Before or after the other inline styles? I'd think they should load after. 

I've also found that styles are now inconsistent, they load in different order depending on the context.

In the front, post editor, and template editor we do:

1. inline global styles
2. theme styles (the ones loaded via _add_editor_style()_ in the editors)

In the site editor we do:

1. theme styles
2. global styles

cc @scruffian @andrewserong I thought we had them all loading in the same order?

Reply:
Sure, we can load them after.